administrator/components/com_privacy/src/Model/ConsentsModel.php

```diff
@@ -105,6 +105,13 @@ protected function getListQuery()
                 ->bind(':state', $state, ParameterType::INTEGER);
         }
 
+        $subject = $this->getState('filter.subject');
+
+        if (!empty($subject)) {
+            $query->where($db->quoteName('a.subject') . ' = :subject')
+                ->bind(':subject', $subject, ParameterType::STRING);
+        }
+        
```

Comment:
```suggestion
        }

```
Fix PHPCS no space in empty line.